lib/pages/inventory.dart

```diff
@@ -1,8 +1,69 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:scibble/models/product.dart';
+import 'package:scibble/redux/inventory/actions.dart';
+import 'package:scibble/redux/store.dart';
+import 'package:scibble/theme/scibble_color.dart';
+import 'package:scibble/widgets/hamburger.dart';
+import 'package:scibble/widgets/scibble_app_bar.dart';
 
 class Inventory extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Inventar');
+    return Scaffold(
+        appBar: ScibbleAppBar(),
+        drawer: HamburgerMenu(),
+        body: StoreConnector<AppState, _InventoryViewModel>(
+          converter: (store) => _InventoryViewModel(store.state.inventory,
+              getInventory: store.dispatch(getInventory(store))),
+          builder: (_, viewModel) => InventoryViewModel(),
+          onInit: (store) => store.dispatch(getInventory(store)),
+        ));
+  }
+}
+
+class InventoryViewModel extends StatelessWidget {
```

Comment:
Might be an idea to abstract this listview into its own widget.
The viewmodel should probably contain an "onClick" which does something for each item. E.g. in this page it sends you to its own edit page, and on the barcode page, it connects the ean to the item.